includes/config.php

```diff
@@ -0,0 +1,16 @@
+<?php
+const SSL_ENABLED = true;
+const BASE_URL = '/test/';
+
+const DB_NAME = 'joobs_v_3_0';
+const DB_USER = 'root';
+const DB_PASSWORD = '';
+const DB_HOST = 'localhost';
+
+const IMAGE_PATH = 'img/company-logo/';
+const DOC_UPLOAD_PATH = 'uploads/';
```

Comment:
Could you please use `define()` instead of `const`

Reply:
Fixed